apiserver/Makefile

```diff
@@ -170,7 +184,7 @@ clean-dev-tools: ## Remove all development tools
 ##@ Testing Setup and Tools
 KIND_CONFIG ?= hack/kind-cluster-config.yaml
 KIND_CLUSTER_NAME ?= ray-api-server-cluster
-OPERATOR_IMAGE_TAG ?= latest
+OPERATOR_IMAGE_TAG ?= nightly
```

Comment:
@tedhtchang this change should probably fix #1350

Reply:
The make deploy-opeator did work with the change.